scripts/create_campaign_jira_issues.py

```diff
@@ -0,0 +1,507 @@
+#!/usr/bin/env python3
+
+from __future__ import annotations
+
+import enum
+import os
+import time
+from dataclasses import dataclass
+from typing import Any, NoReturn
+
+import requests
+import typer
+from dotenv import load_dotenv
+from requests.auth import HTTPBasicAuth
+from rich.console import Console
+from rich.progress import Progress, SpinnerColumn, TextColumn
+from rich.table import Table
+
+load_dotenv()
+
+console = Console()
+err_console = Console(stderr=True)
+
+JIRA_URL = "https://{YOUR TEAM}.atlassian.net"
```

Comment:
@davidbloss - this would be one place we might have to have configurable from the outside